kolibri_explore_plugin/assets/src/views/ExploreIndex.vue

```diff
@@ -11,13 +11,14 @@
         <CollectionSelectionModal
           :visible="collectionModalVisible"
           :grade="grade"
-          :collections="collections"
+          :collections="gradeCollections"
           @downloadCollection="downloadCollection"
           @goBack="visibleModal = 'grade'"
         />
         <InstallContentModal
           :visible="contentModalVisible"
           :collection="downloadingCollection"
+          :grade="grade"
```

Comment:
I feel like "grade" is an awkward word to use here since it's very specific to our current implementation, but at the same time the alternatives I can think of ("group", "category") would be terrrible and confusing, so :shrug: